tap_exacttarget/state.py

```diff
@@ -19,13 +19,14 @@
 })
 
 
-def get_last_record_value_for_table(state, table):
+def get_last_record_value_for_table(state, table, config):
```

Comment:
Why this new parameter being added? Please add some comments to the changes done

Reply:
Added comment